sensor/test/sensor/sen0177/sensor-data-transformer.js

```diff
@@ -0,0 +1,43 @@
+let chai = require('chai');
+let expect = chai.expect;
+
+describe('SensorDataTransformer', () => {
+  const SensorDataTransformer = require('../../../services/sensor/sen0177/sensor-data-transformer');
+
+  it('Should properly convert integer values', (done) => {
+    let parseInt = function (arr) {
```

Comment:
```let parseInt = SensorDataTransformer.parseInt32```

Basically remeber that every time that you see
```js
let variable = function (arr) {
   someMethod(arr)
}}}
```
could be simplified to (almost always)
```  let variable = someMethod  ```